docs/source/support/faq.rst

```diff
@@ -96,6 +86,16 @@ Q: How do I integrate nucypher if my application is not written in Python?
 
 Nucypher also provides web integration. See :ref:`web-development`.
 
+.. _faq-testnet:
+
+Q: Are there testnets that I can use?
```

Comment:
This is subject to near-term change as we stabilize the new release across testnet nodes.

Reply:
✔️